supply-chain/catalog/trivy-scanning.hbs.md

```diff
@@ -0,0 +1,64 @@
+# Trivy Scanning Component
+
+## Description
+
+This document describes the Trivy Scanning Component for Tanzu Supply Chains. The Trivy Component provides out of the box trivy scanning that invokes [Supply Chain Security Tools - Scan 2.0](../../scst-scan/scan-2-0.hbs.md) in a Tanzu Supply Chain. See aquasecurity trivy [docs](https://github.com/aquasecurity/trivy) for more details on the trivy scanner.
+
+## API
+
+Component input: `image` is an input from the buildpack component
+
+Workload input: `spec.registry` is where the scan results are uploaded
+
+Workload input: `spec.source` is for the source component to retrieve source from git
```

Comment:
Might be a knowledge gap from my part: Why do we need git source for trivy image scanning?

Reply:
Thanks for raising this. I've been back and forth on this. I think something here needs to be revised.

The `spec.source` gets used in two places:
1) The input to our trivy supply chain component is `git` (shown [here](https://gitlab.eng.vmware.com/tanzu-application-platform/secure-tools/scanning/-/blob/main/packages/trivy.app-scanning.component.apps.tanzu.vmware.com/config/trivy-supply-chain-component.yaml#L529)) where `git` comes from `spec.source` that requires `git`. This `git` gets used to produce a `correlation-id` for the IVS. 
2) The `spec.source.git` gets used for the source component for git cloning. This probably belongs in the `Source Component` docs and not ours.

Technically the trivy supply chain component is using this `spec.source.git` for the `correlation-id` but I don't know if I should be mentioning a `correlation-id` here.

Long story short:
* My inclination now is to remove this line because it is confusing and I'll have to address this in a next docs PR which I'm working on now.

Reply:
ah makes sense. thanks!